knowledge-base/export-websocket-traffic.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Exporting WebSocket Traffic
+description: Learn how to export WebSocket traffic using Fiddler Everywhere.
+type: how-to
+page_title: Export WebSocket Traffic as Plain Text | Fiddler Everywhere
+slug: export-websocket-traffic-plain-text
+tags: fiddler-everywhere, export, websocket, plain-text
+res_type: kb
+---
+
+## Environment
+
+| Property | Value |
+|----------|-------|
+| Product  | Fiddler Everywhere |
+
+## Description
+
+To export WebSocket traffic using Fiddler Everywhere, use one of the following options:
+
+- Save or export the entire WebSocket sessions through the **Save** and **Export** options in the Live Traffic grid context menu. This allows you to preserve the whole WebSocket session and reload it later for further inspection.
+
+- Load the **Messages inspector**, and use the **Copy all content to clipboard** option from the toolbar. This will copy all messages alongside their technical data in plain text format.
```

Comment:
```suggestion
- Load the **Messages inspector**, and use the **Copy all content to clipboard** option from the toolbar icons. This will copy all messages alongside their technical data in plain text format.
```
I added the icon mention because initially, I was looking at the context menu options and couldn't find it. Probably I have a stronger association that links the word "option" with a context menu, not with the icons in the toolbar.